test/player-full.html

```diff
@@ -7,6 +7,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.min.css">
     <link rel="stylesheet" href="../src/css/mediaelementplayer.css">
+    <link rel="stylesheet" href="../src/css/mediaelementplayer-legacy.css">    
```

Comment:
@johndyer Do we need this file here? I removed it for testing purposes and doesn't affect anything